### DISCUSSION TITLE
Is there any safe method for adding firewall exceptions to merge modules?

### DISCUSSION THREAD

**Discussion by AlasdairWJ** [wixtoolset/issues#6518] [Questions] [answered]
Question is in the title mostly, I'd like to be able to distribute merge modules that includes firewall exceptions safely, but as I understand it, it isn't possible to do this without risking conflicts. Is there any way of achieving this? (The only solutions I know of currently is to use wixlibs rather than merge modules)

The relavent warning is:
warning LGHT1072: Merge modules should not contain the 'Error' table because all merge conflicts cannot avoided.  However, this warning can be suppressed if all of the consumers of the Merge Module agree to not duplicate identifiers in the 'Error' table.

**Comment by robmen** [comment, marked as answer]
This is one of the few cases where it is better to Ignore the ICE warning. It is unlikely that consumers of your Merge Module will have error numbers that collide with the WiX Toolset's span of error numbers.

**Reply by AlasdairWJ** [reply]
Thanks for the speedy response, much appreciated!
